calico_node/Makefile

```diff
@@ -388,6 +390,41 @@ run-etcd-host:
 	--advertise-client-urls "http://$(LOCAL_IP_ENV):2379,http://127.0.0.1:2379" \
 	--listen-client-urls "http://0.0.0.0:2379"
 
+## Kubernetes apiserver used for tests
+run-k8s-apiserver: stop-k8s-apiserver run-etcd vendor
```

Comment:
I don't think this should depend on vendor.

Reply:
I think we need to, if we're pulling in the CRDs from libcalico-go in the vendor dir this will fail if it doesn't exist.

Reply:
Ah yeah, right.